File: app/src/main/java/org/evcs/android/model/user/UserRequest.java
package org.evcs.android.model.user;

public class UserRequest {

    private String name;
    private final String email;
    private final String password;

    public UserRequest(String email, String password) {
        this.email = email;
        this.password = password;
    }

    public UserRequest(String name, String email, String password) {
        this.name = name;
        this.email = email;
        this.password = password;
    }
}

File: app/src/main/java/org/evcs/android/model/user/UserRequest.kt
package org.evcs.android.model.user

class UserRequest {
    private var name: String? = null
    private val email: String
    private val password: String

    constructor(email: String, password: String) {
        this.email = email
        this.password = password
    }

    constructor(name: String?, email: String, password: String) {
        this.name = name
        this.email = email
        this.password = password
    }
}